branding/spacewalk-branding.spec

```diff
@@ -109,6 +109,7 @@ jar -cf java-branding.jar -C java/code/src com
 ln -s /srv/www/htdocs/css/bootstrap css/bootstrap
 ln -s /srv/www/htdocs/css/patternfly1 css/patternfly1
 lessc css/spacewalk.less > css/spacewalk.css
+lessc css/uyuni/spacewalk.less > css/uyuni/spacewalk.css
```

Comment:
I don't think this is going to be enough, what about the file package list? :-)

It only does:
```
%dir %{wwwdocroot}/css
%{wwwdocroot}/css/*.css
```
so neither the `uyuni` directory or the css files inside will be packaged.

FMPOV you need to add:
```
%dir %{wwwdocroot}/css/uyuni
%{wwwdocroot}/css/uyuni/*.css
```

Reply:
BTW: do both SUSE Manager and Uyuni require both sets of CSS files at the same time?

Otherwise we should consider packaging those for SUSE Manager when OS is SLE, and those for Uyuni when OS is openSUSE.